Localization/en-US.hjson

```diff
@@ -0,0 +1,222 @@
+			SandstoneBrickWall: Sand Brick Wall
+			SandstoneSlab: Sand Slab
+			SandstoneSlabWall: Sand Slab Wall
+		}
+
+		Config: {
+			ServerConfig: General Settings
+			ServerBlocked: Only the server owner can change these mod settings.
+
+			GladiatorArmorTweak: {
+				$parentVal: Gladiator Armor Tweaks
+				Desc:
+					'''
+					Defense increased from 2/3/2 to 5/6/5
+					Set bonus: +15% crit chance
+					'''
+			}
+
+			ObsidianArmorTweak: {
+				$parentVal: Obsidian Armor Tweaks
+				Desc:
+					'''
+					+3% ranged crit chance for each piece
+					Set bonus: +10% move speed and shadow trail
+					'''
+			}
+
+			MeteorArmorDamageTweak: {
+				$parentVal: Meteor Armor Damage Tweaks
+				Desc: Remove magic damage boosts from Meteor Armor.
+			}
+
+			EskimoArmorDropTweak: {
+				$parentVal: Eskimo Armor Drop Rate Increase
+				Desc: 3% chance to drop Eskimo Armor instead of 0.67% (twice as much in Expert Mode).
+			}
+
+			EskimoArmorTweak: {
+				$parentVal: Eskimo Armor Tweaks
+				Desc:
+					'''
+					Defense increased, immunity to cold water
+					Set bonus: Warmth buff, immunity to Frostburn, Chilled and Frozen
+					'''
+			}
+
+			RainArmorTweak: Rain Armor is now Vanity
+
+			HammerTweaks: {
+				$parentVal: Hammer Tweaks
+				Desc: Rebalances a few hammers/hamaxes.
+			}
+
+			SwatHelmetTweak: {
+				$parentVal: SWAT Helmet Tweaks
+				Desc:
+					'''
+					+14 defense, +15% ranged damage (all types), +5% ranged crit chance
+					If Miscellania is installed, equipping a SWAT Helmet and a Reinforced Vest grants a set bonus of:
+					  Increase ranged damage by 20%
+					  Reduces damage taken by 25%
+					  20% chance to not consume ammo
+					'''
+			}
+
+			FishBowlTweak: {
+				$parentVal: Fish Bowl Tweak
+				Desc: Gives the Fish Bowl +1 defense.
+			}
+
+			BoneBlockFix: {
+				$parentVal: Bone Block Fix
+				Desc: Bone Blocks drop raw Bones when mined.
+			}
+
+			GoldCritterDropTweak: {
+				$parentVal: Gold Critters Drop Coins
+				Desc: Gold critters drop 1 Gold when killed (still sell for 5 Gold when sold).
+			}
+
+			WhoopieCushionTweak: Easter Egg
+
+			VikingHelmetTweak: {
+				$parentVal: Viking Helmet Tweaks
+				Desc: Viking Helmet gives +5% damage when used with Iron or Lead armor.
+			}
+
+			CactusArmorTweak: {
+				$parentVal: Cactus Armor Tweaks
+				Desc: Changes the set bonus of cactus to 25% chance of thorns.
+			}
+
+			MeteorArmorDefenseTweak: {
+				$parentVal: Meteor Armor Defense Tweaks
+				Desc: Defense decreased from 5/6/5 to 4/5/4.
+			}
+
+			PharaohSetTweak: {
+				$parentVal: Pharaoh Set Tweaks
+				Desc: Changes Pharaoh's vanity set to Summoner armor.
+			}
+
+			CrimsonArmorTweak: {
+				$parentVal: Crimson Set Tweak
+				Desc: Crimson Armor now only increases melee damage instead of all damage.
+			}
+
+			UndeadMinerDropRateTweak: {
+				$parentVal: Undead Miner Drop Rate Tweak
+				Desc: Changes the drop rate of mining clothes to a 1/4 chance (1/3 in Expert mode).
+			}
```

Comment:
Re-Logic buffed this, so this tweak might not be necessary.

Reply:
The drop rate is basically the same as ours but they did it without the expert mode change.
do we remove the normal mode change or both?

Reply:
Same as the fishing pole tweak, I'd like to remove it entirely.